core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -3102,6 +3107,9 @@ def _train_multi_fold_parallel(
                 # Re-schedule jobs
                 unfinished_job_refs += distributed_manager.schedule_jobs()
 
+        # FIXME: this might be slightly off due to less training data used for the bagging fit (depending on the validation method).
+        self._refit_mem_cache = distributed_manager.refit_model_child_mem_estimate_cache.copy()
```

Comment:
I don't think this will work properly for multi-layer stacking, because you will overwrite the previous layer's mem cache.

Reply:
Should also be probably using the DAG to store this info

Reply:
Ture, will need to fix this (by extending the dict). 

Yes, we can get it as a return value as well. Looking into it.